db/schema.rb

```diff
@@ -997,8 +1002,13 @@
     t.datetime "reset_password_sent_at"
     t.string   "api_key",                :limit => 40
     t.integer  "enterprise_limit",                     :default => 1, :null => false
+    t.string   "confirmation_token"
+    t.datetime "confirmed_at"
+    t.datetime "confirmation_sent_at"
+    t.string   "unconfirmed_email"
   end
 
+  add_index "spree_users", ["confirmation_token"], :name => "index_spree_users_on_confirmation_token", :unique => true
```

Comment:
@sauloperez this index is [specified by Devise](https://github.com/plataformatec/devise/wiki/How-To:-Add-:confirmable-to-Users), it's used in exactly the context you mention: we need to do a lookup on the token to work out which user is trying to confirm their email address.